src/components/Table/GridTable.stories.tsx

```diff
@@ -724,6 +730,99 @@ export function CustomEmptyCell() {
   );
 }
 
+type BeamData = {
+  id: string;
+  favorite: boolean;
+  status: string;
+  commitmentName: string;
+  tradeCategories: string[];
+  location: string;
+  date: string;
+  priceInCents: number;
+};
+type BeamRow = SimpleHeaderAndDataOf<BeamData>;
+export function StyleBeam() {
+  const locations: HasIdAndName<string>[] = [
+    { id: "l:1", name: "Living Room" },
+    { id: "l:2", name: "Great Room" },
+  ];
+
+  const selectCol = selectColumn<BeamRow>({
+    header: () => ({ content: <Checkbox label="Label" onChange={noop} checkboxOnly /> }),
+    data: () => ({ content: <Checkbox label="Label" onChange={noop} checkboxOnly /> }),
+  });
+  const favCol = column<BeamRow>({
+    header: () => ({ content: "" }),
+    data: ({ favorite }) => ({
+      content: <Icon icon="star" color={favorite ? Palette.Gray700 : Palette.Gray300} />,
+      sortValue: favorite ? 0 : 1,
+    }),
+    // Defining `w: 56px` to accommodate for the `27px` wide checkbox and `16px` of padding on either side.
+    w: "56px",
+  });
+  const statusCol = column<BeamRow>({
+    header: "Status",
+    data: ({ status }) => ({ content: <Tag text={status} type="success" />, sortValue: status }),
+  });
+  const nameCol = column<BeamRow>({ header: "Commitment Name", data: ({ commitmentName }) => commitmentName });
+  const tradeCol = column<BeamRow>({
+    header: "Trade Categories",
+    data: ({ tradeCategories }) => ({
+      content: () => (
+        <div css={Css.df.gap1.$}>
+          {tradeCategories.sort().map((tc) => (
+            <Chip text={tc} />
+          ))}
+        </div>
+      ),
+      sortValue: tradeCategories.sort().join(" "),
+    }),
+  });
+  const dateCol = dateColumn<BeamRow>({ header: "Date", data: ({ date }) => date });
+  const locationCol = column<BeamRow>({
+    header: "Location",
+    data: (row) => ({
+      content: <SelectField value={row.location} onSelect={noop} options={locations} label="Location" hideLabel />,
```

Comment:
Nudge for GridTable to use a PresentationProvider to auto-`hideLabel` all of these :-)

Reply:
Great point, this is already done... I forgot!

Reply:
Hot damn! :-D